_tests/test/core/event_handler_test.dart

```diff
@@ -138,16 +138,16 @@ void main() {
     });
   });
 
-  test('should support static methods tear-offs for events', () async {
-    final testBed = NgTestBed<TestStaticMethods>(
-      ng.createTestStaticMethodsFactory(),
-    );
-    final fixture = await testBed.create();
-    TestStaticMethods.overrideDoCapture = expectAsync0(() {});
-    await fixture.update((_) {
-      fixture.rootElement.querySelector('button')!.click();
-    });
-  });
+  // test('should support static methods tear-offs for events', () async {
```

Comment:
I'd recommend using [`skip: 'See https://github.com/angulardart-community/angular/issues/3'`](https://pub.dev/packages/test#skipping-tests) instead of commenting out, so we will actually remember it.